managed/src/main/java/com/yugabyte/yw/controllers/EncryptionAtRestController.java

```diff
@@ -36,65 +36,59 @@
 import java.util.stream.Collectors;
 
 public class EncryptionAtRestController extends AuthenticatedController {
-    public static final Logger LOG = LoggerFactory.getLogger(EncryptionAtRestController.class);
-
-    @Inject
-    EncryptionAtRestManager keyManager;
-
-    @Inject
-    Commissioner commissioner;
-
-    public Result createKMSConfig(UUID customerUUID, String keyProvider) {
-        LOG.info(String.format(
-                "Creating KMS configuration for customer %s with %s",
-                customerUUID.toString(),
-                keyProvider
-        ));
-        Customer customer = Customer.getOrBadRequest(customerUUID);
-        try {
-          TaskType taskType = TaskType.CreateKMSConfig;
-          ObjectNode formData = (ObjectNode) request().body().asJson();
-          KMSConfigTaskParams taskParams = new KMSConfigTaskParams();
-          taskParams.kmsProvider = Enum.valueOf(KeyProvider.class, keyProvider);
-          taskParams.providerConfig = formData;
-          taskParams.customerUUID = customerUUID;
-          taskParams.kmsConfigName = formData.get("name").asText();
-          formData.remove("name");
-          UUID taskUUID = commissioner.submit(taskType, taskParams);
-          LOG.info("Submitted create KMS config for {}, task uuid = {}.", customerUUID, taskUUID);
-          // Add this task uuid to the user universe.
-          CustomerTask.create(customer,
-            customerUUID,
-            taskUUID,
-            CustomerTask.TargetType.KMSConfiguration,
-            CustomerTask.TaskType.Create,
-            taskParams.getName());
-          LOG.info("Saved task uuid " + taskUUID + " in customer tasks table for customer: " +
-            customerUUID);
-
-          auditService().createAuditEntry(ctx(), request(), formData);
-          return new YWResults.YWTask(taskUUID).asResult();
-        } catch (Exception e) {
-          throw new YWServiceException(BAD_REQUEST, e.getMessage());
-        }
+  public static final Logger LOG = LoggerFactory.getLogger(EncryptionAtRestController.class);
+
+  @Inject EncryptionAtRestManager keyManager;
+
+  @Inject Commissioner commissioner;
+
+  public Result createKMSConfig(UUID customerUUID, String keyProvider) {
+    LOG.info(
+        String.format(
+            "Creating KMS configuration for customer %s with %s",
+            customerUUID.toString(), keyProvider));
+    Customer customer = Customer.getOrBadRequest(customerUUID);
+    try {
+      TaskType taskType = TaskType.CreateKMSConfig;
+      ObjectNode formData = (ObjectNode) request().body().asJson();
+      KMSConfigTaskParams taskParams = new KMSConfigTaskParams();
+      taskParams.kmsProvider = Enum.valueOf(KeyProvider.class, keyProvider);
+      taskParams.providerConfig = formData;
+      taskParams.customerUUID = customerUUID;
+      taskParams.kmsConfigName = formData.get("name").asText();
+      formData.remove("name");
```

Comment:
try catch should end here. Do not include line 60. Errors beyond this point are not due to bad request